Change dash image (#285)
farm-ng/amiga-dev-kit: website/docs/dashboard/dashboard_user_guide.md

title: Dashboard User Guide
---
:::caution
This guide is subject to change with each new update to the Dashboard.

> The latest version is Dashboard Firmware release `v0.4.2`.

Reviewer: 0.4.2 is an internal release, and is not publicly available

```suggestion
The latest version is Dashboard Firmware release `v0.4.0`.
```

allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
allowfullscreen></iframe>

## Start Page

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/fccbf6aa-95ae-41cb-bb65-fb816451c011"/>

This is the start page on the dashboard.
It will appear when you first turn on your Amiga.
Here is where you can change the display's language.
You may choose between English, Spanish, and French.
Pressing the START button will take you to the home screen.

## Home Screen

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/954a66d5-2155-48f1-9830-25428374fa2a"/>

This is the home screen of the dashboard. In the center sits the spedometer,
 where you can switch between metric and standard units and adjust to the travel speed of your Amiga.

> git  On the right side of the screen you will see several icons displaying:

Reviewer: ```suggestion
On the right side of the screen you will see several icons displaying:
```

- Battery Level
- Average motor temperature
- Connection to Pendant
- Preview of each motor's health

You can always come back to this screen by pressing the upper left icon on the dashboard.

## Motor Status Screen

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/d0cc17f2-3cdf-49c5-ab02-e412cbec56bd"/>

In this screen you can see the control state of your Amiga:

- Active
- Cruise
- Auto Ready/Active
- E-Stopped

>  You will also see details on each motor's:

Reviewer: ```suggestion
 You will also see details on each motor / motor controller's:
```

> - Voltage
> - RPM
> - Temperature
> - Current draw for each motor/controller

Reviewer: ```suggestion
- Voltage
- RPM
- Temperature
- Current draw
```

> Lastly, CANbus states appear on the right side of the display.

Reviewer: ```suggestion
Lastly, CAN bus states appear on the right side of the display.
```

> Follow this link for an indepth description of your Amiga's
> [Canbus Service](https://amiga.farm-ng.com/docs/concepts/canbus_service/)

Reviewer: That link is for the brain interacting with the can bus. It's not really applicable to the CAN states IMO 

```suggestion
```

## Auto Control Screen

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/9a8dcddf-cb5d-4e3c-95e0-0224f521ae6d"/>

> If you have an [Intelligence Kit](/docs/intelligence-kit/overview-intel)
>  mounted on your Amiga and have some tracks recorded in
> [Autoplot](https://amiga.farm-ng.com/docs/apps/autoplot_app/), it is here where autonomous control
> comes to life.
> From this screen, you will be able to:

Reviewer: ```suggestion
If you have an [Intelligence Kit](/docs/intelligence-kit/overview-intel)
mounted on your Amiga and have some tracks recorded in
[Autoplot](https://amiga.farm-ng.com/docs/apps/autoplot_app/), 
activating the auto mode on this screen is what brings autonomous control into action.

From this screen you will be able to:
```

- Enable and disable Auto Control
- Monitor the Auto Control state
- Monitor the commanded & measured speed of your Amiga

> You can use other micro-controllers to command your Amiga.
> For an example of how to do so, please follow the instructions in this [FPV example](/docs/examples/FPV/)

Reviewer: ```suggestion
You can also use micro-controllers to command your Amiga, like our [Microcontroller Kit](/docs/mcu_kit/).
For an example of how to do so, please follow the instructions in this [Hello Main Loop example](/docs/examples/hello_main_loop/) or this [FPV example](/docs/examples/FPV/)
```

A purple square indicates the need to re-calibrate your pendant.

:::caution
If your Amiga begins to move on its own at start up, chances are that your pendant needs to be calibrated.
With the above image in mind (purple square around the white dot), your Amiga would move forwards
and towards the right at start up.
:::

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/8bc4980b-d6f4-445b-b772-0b7e46b7eb1a"/>

Pressing the Pendant image on the right will show the latest button configurations.

:::tip
The arrows (up/down/left/right) in the center of the pendant are used to select and engage with the H-Bridge(s).
:::

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/0f880f2a-3958-4066-989d-d68b68547665"/>

These are the button configurations for those of you who purchased a Kartech control.

### H-bridge settings

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bd24caf0-01c7-4647-b6ba-9ce6c55c47aa"/>

An H bridge circuit allows you to apply a voltage across a load (i.e. a linear actuator)
in either direction. We currently support up to four h bridge modules on the Amiga.

> They can be engaged momertarily or lached.
> The `up` - `down` arrows are used to `Active` the H bridge

Reviewer: ```suggestion
They can be engaged momentarily or latched.
The `up` - `down` arrows are used to control the selected H bridge device(s)
```

and the `left` - `right` arrow keys are used to select between H-bridge devices.

> As of this writing only one H-bridge device can be actuated at a time.

Reviewer: I thought we can couple in the released fw? If not, we can say starting with `v0.5.0` (unless there is a bug that needs to be fixed?)
```suggestion
Two H-bridge devices (0 & 1 or 2 & 3) can be coupled with the **coupler** button and actuated together.
```

The PTO (power take-off) screen allows you to select between devices (PTO0 &PTO1), fine tune their
settings and monitor their performance.
We currently support up to two PTO's running at the same time.

When you connect a PTO device, you will have access to the following features:

- A slider control that enables you to increade or decrease the PTO's RPM.
- Directional control by means of a toggle switch on the touchscreen.

On the left side of the screen you will see a graph displaying the current PTO set-point &
measured rpm values.

:::tip
The default, min, & max values for the RPM slider can be configured on the settings page.
The PTO gear ratio can also be configured on the settings page.
:::

<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/e08dd594-e229-4820-904c-9e4f9aeb2bc1"/>

Occasionally a message stating that no PTO device has been detected will be accompanied by the
following image.
Inspect all cables and make sure they are properly connected.
<img src="https://github.com/farm-ng/amiga-dev-kit/assets/133177230/bf3d2a67-d10b-41ca-b35b-89fc0f3a1199"/>

If you expect a PTO device but it is not showing up,

>  please reach out to us via farm-ng's

Reviewer: ```suggestion
please reach out to us via farm-ng's
```
